src/puppet-padpro.ts

```diff
@@ -987,6 +989,24 @@ export class PuppetPadpro extends Puppet {
     }
   }
 
+  public async messageMiniProgram (messageId: string): Promise<MiniProgramPayload> {
+
+    const rawPayload = await this.messageRawPayload(messageId)
+    const payload = await this.messagePayload(messageId)
+
+    if (payload.type !== MessageType.MiniProgram) {
+      throw new Error('Can not get miniProgram from non miniProgram payload')
+    } else {
+      const appPayload = await appMessageParser(rawPayload)
+      if (appPayload) {
+        return {
```

Comment:
What's the reason returning empty object here?

Reply:
Is messageForward work well?
I tested with text , using 'msg.forward(contact)', it's not work!
I confirmed that code was step into messageForward->this.messageSendText
receiver:
```json
{ contactId: 'wxid_gjg27bv08vu612', roomId: undefined }
```
payload:
```json
{ id: '1097670511',
  timestamp: 1564021102,
  type: 7,
  fromId: 'wxid_jy0q2q63aggm21',
  mentionIdList: undefined,
  roomId: undefined,
  text: 'hello world',
  toId: 'wxid_gjg27bv08vu612' }
```
![image](https://user-images.githubusercontent.com/3889726/61841308-e9284980-aec6-11e9-81f3-55884c0d75b0.png)